README.md

```diff
@@ -0,0 +1,50 @@
+# Auto Print
+
+Google Apps Script microservice for printing documents via PrintNode and sending faxes via SFax
+
+---
+
+## Development Environment Setup
+
+> **WARNING:** always pull from production before making any changes since push/pull is destructive and will overwrite local/remote changes without warning!
+
+Code is pulled using clasp which requires nodeJS to be installed:
+
+
+1. Install clasp globally
+ ```zsh
+ npm i -g @google/clasp
+ ```
+
+2. Clone this repo
+ ```zsh
+ git clone git@github.com:dscsa/auto-print.git
+ cd auto-print
+ ```
+
+3. Login to clasp. This should open a browser tab prompting auth via Google
```

Comment:
I would maybe just clarify that you are logging into the print@sirum.org account here. I know its mentioned further down but maybe it saves someone an extra step.

Reply:
Good point, thanks!